geomesa-core/src/main/scala/org/locationtech/geomesa/core/stats/StatWriter.scala

```diff
@@ -60,7 +61,7 @@ object StatWriter extends Runnable with Logging {
 
   private val writeDelayMillis = 1000
 
-  private val batchWriterConfig = new BatchWriterConfig().setMaxMemory(10000L).setMaxWriteThreads(5)
+  private val batchWriterConfig = GeoMesaBatchWriterConfig().setMaxMemory(10000L).setMaxWriteThreads(5)
```

Comment:
Maybe there should be GeoMesaBatchWriter.get{Feature,Stats,Metadata}Config methods?  (The last two could share.)